test/io/test_io.py

```diff
@@ -555,13 +555,15 @@ def sleep(i=i):
         assert delta > 1 and delta < 1.5
 
 
-def test_pool_acquisition_timeout(defaultenv, metapostgrest):
+@pytest.mark.parametrize("level", ["crit", "error", "warn", "info"])
```

Comment:
Nice! :100:

Reply:
Yes, Codecov is not complaining anymore.